test/intl402/Temporal/TimeZone/etc-timezone.js

```diff
@@ -40,26 +47,35 @@ assert.sameValue(
   let tz = "Etc/GMT+0" + n;
   assert.throws(
     RangeError,
-    () => { Temporal.TimeZone.from(tz); },
+    () => { new Temporal.TimeZone(tz); },
     tz + " is an invalid timezone"
     );
 });
 
-// Etc/GMT+0" through "Etc/GMT+9" are OK
+// Etc/GMT+0" through "Etc/GMT+12" are OK
 
-// zero is handled in its own way:
+// zero is handled in its own way (say "UTC" rather than "GMT"):
 assert.sameValue(
-  Temporal.TimeZone.from("Etc/GMT+0").toString(),
+  (new Temporal.TimeZone("Etc/GMT+0")).toString(),
   "UTC", // if the offset is +0, we say "UTC" rather than "GMT"
   "Etc/GMT+0 is a valid timezone"
 );
 
 [1,2,3,4,5,6,7,8,9, 10, 11, 12].forEach((n) => {
```

Comment:
Can we correct the variable spacing around commas here?

Reply:
done